plugins/task/demotasks/demotasks.php

```diff
@@ -68,6 +74,60 @@ public static function getSubscribedEvents(): array
 		];
 	}
 
+	/**
+	 * Sample resumable task.
+	 *
+	 * Whether the task will resume is random. There's a 40% chance of finishing every time it runs.
+	 *
+	 * You can use this as a template to create long running tasks which can detect an impeding
```

Comment:
```suggestion
	 * You can use this as a template to create long running tasks which can detect an impending
```
Made a suggestion so you can apply with one click. Thanks @brianteeman for having found it.

Reply:
From my phone it wasn't easy to tell.